### PR TITLE
New: Update all wiki links to point to the Servarr Wiki

### LINKED ISSUE #653: Pull Sonarr commit 'All Wiki links now use the consolidated Servarr wiki'
Pull https://github.com/Sonarr/Sonarr/commit/e733529dc3e0c2fe9fbed86dbee6c295627d44f5

```
git cherry-pick -ex e733529dc3e0c2fe9fbed86dbee6c295627d44f5
```

This commit has 16 conflict(s)

You can ask me to `close all`, `open pr here` or `open pr everywhere`

### LINKED ISSUE #656: Pull Sonarr commit 'Fixed health check wiki link unit tests'
Pull https://github.com/Sonarr/Sonarr/commit/b84f84ad0a0bab65580e72eb68926dd82bc9fa66

```
git cherry-pick -ex b84f84ad0a0bab65580e72eb68926dd82bc9fa66
```

This commit has 1 conflict(s)

You can ask me to `close all`, `open pr here` or `open pr everywhere`

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
 NO

#### Description
Update links to use Servarr Wiki

#### Todos
- [X] Tests
- [ ] Wiki Updates - assigned to Drazz
- [x] Review and Update Health Check Wiki Links
#### Issues Fixed or Closed by this PR

* Fixes #656
* Fixes #653

### CODE REVIEW

**Comment by Qstick** [comment]
Wanna update all the fragments in here

**Comment by bakerboy448** [comment]
Quick rebase to clean up commits after mega PR will be needed

This has been rebased on the sonarr-pulls-test branch

**Comment by Qstick** [comment]
@bakerboy448 rebase on develop please :)

File: src/NzbDrone.Core/HealthCheck/Checks/DotnetVersionCheck.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+using System;
```
Comment (Qstick) [inline src/NzbDrone.Core/HealthCheck/Checks/DotnetVersionCheck.cs]: Why'd you add this back?

**Review by Qstick — CHANGES_REQUESTED** [review]

File: src/Readarr.Api.V1/ProviderResource.cs [hunk 2]
```diff
@@ -38,8 +38,7 @@ public virtual TProviderResource ToResource(TProviderDefinition definition)
                 Tags = definition.Tags,
                 Fields = SchemaBuilder.ToSchema(definition.Settings),
 
-                InfoLink = string.Format("https://github.com/Readarr/Readarr/wiki/Supported-{0}#{1}",
-                    typeof(TProviderResource).Name.Replace("Resource", "s"),
+                InfoLink = string.Format("https://wiki.servarr.com/Readarr_Supported-{0}",
```
Comment (Qstick) [inline src/Readarr.Api.V1/ProviderResource.cs]: should this be `"https://wiki.servarr.com/Readarr_Supported_{0}"`

**Review by Qstick — APPROVED** [review]

**Pull request merged**